app/components/op_turbo/frame_wrapper_component.rb

```diff
@@ -0,0 +1,35 @@
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2023 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+module OpTurbo
+  class FrameWrapperComponent < ApplicationComponent
```

Comment:
I would remove the `Wrapper` part of the name.

Reply:
Any reason why? I imagine we want to be consistent with `OpTurbo::StreamWrapperComponent`

Reply:
True, I would remove `Wrapper` from `OpTurbo::StreamWrapperComponent` as well. 
When I read `OpTurbo::FrameWrapperComponent` and `OpTurbo::StreamWrapperComponent`, I'm thinking that they may be doing something special as they "wrap" something. But in fact no, they are just acting as expected.

I feel like "wrapper" is not adding any information here and the code reads equally well without it (and even better in my opinion). For instance when we are building a form, we are not using `form_wrapper_tag`, `label_wrapper_tag`, `input_wrapper_tag`, and `submit_wrapper_tag`. We are using `form_tag`, `label_tag`, `input_tag`, and `submit_tag` because they do what they announce: they render a tag of a particular kind. I think it is similar in this case: naming it `OpTurbo::FrameComponent` and `OpTurbo::StreamComponent` would make the name shorter while keeping the same intent.

Reply:
That being said, it's ok for me if we keep the `Wrapper` part in the name.

Reply:
Took your up recommendation @cbliard 👍🏾 extracted to a separate PR https://github.com/opf/openproject/pull/13932